advance_ai_agents/deep_researcher_agent/.env.example

```
@@ -0,0 +1,6 @@
+﻿# deep_researcher_agent Environment Configuration
```

dotenv-linter (4.0.0)

[warning] [IncorrectDelimiter] The ﻿# deep_researcher_agent Environment Configuration key has incorrect delimiter

(IncorrectDelimiter)